_posts/2021-03-15-micrometer-prometheus-openshift.adoc

```diff
@@ -183,12 +183,15 @@ spec:
   endpoints:
   - interval: 30s
     targetPort: 8080
+    path: /q/metrics
     scheme: http
   selector:
     matchLabels:
       app-with-metrics: 'quarkus-app'
 ----
 
+The path `/q/metrics` is the default metrics endpoint in Quarkus. We can configure it using the property `quarkus.micrometer.export.prometheus.path`
```

Comment:
```suggestion
The path `/q/metrics` is the default metrics endpoint in Quarkus. 

We could configure the `/metrics` endpoint Prometheus expects in `application.properties` instead of adding the path attribute above using the following property: `quarkus.micrometer.export.prometheus.path=/metrics`.
```

Reply:
Maybe this is too much.. your call. ;)